beesdoo_shift/models/res_partner.py

```diff
@@ -195,4 +195,26 @@ def _update_shifts_on_subscribed_task_tmpl(
                 now=datetime.now(),
             )
 
-    # TODO access right + vue on res.partner
+    def get_next_shifts(self):
+        """
+        Get the shifts of the user between now and an end_date,
+        with end_date = 4 weeks * regular_next_shift_limit
+        :return: beesdoo.shift.shift list
+        """
+        regular_next_shift_limit = int(
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("beesdoo_shift.regular_next_shift_limit")
+        )
+        nb_days = 28 * regular_next_shift_limit
```

Comment:
In a general comment, we should improve this and not hardcode `28` anymore. But it's not related to this PR.